packages/camera/example/ios/Runner/Info.plist

```diff
@@ -22,8 +20,16 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>LSApplicationCategoryType</key>
+	<string></string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSCameraUsageDescription</key>
+	<string>Can I use the camera please? Only for demo purpose of the app</string>
+	<key>NSMicrophoneUsageDescription</key>
+	<string>Only for demo purpose of the app</string>
+	<key>UIFileSharingEnabled</key>
```

Comment:
`UIFileSharingEnabled` is not needed but helps you download the video via iTunes in the current version. I'd call it 'work in progress', as the correct method would be to save the video / image created to the device's camera roll.
We can remove this permission to keep the plugin simple.

Reply:
With respect to UIFileSharingEnabled, we need this permission in order to be able to inspect generated file easily with iTunes. And this is important at the moment as we don't preview the video. There are other solutions, like storing files in the camera roll but that may be app specific. I think this can be removed later on once we add better support for video/images preview. Let us know what do you think?

Reply:
I made a pull request on your repo adding a video preview. Using something like that we would not need to bother with iTunes. https://github.com/Tengio/plugins/pull/3

Reply:
We merged it. IT's looking good :)